libraries/remote_management.rb

```diff
@@ -0,0 +1,40 @@
+include Chef::Mixin::ShellOut
+
+module MacOS
+  class RemoteManagement
+    class << self
+      def activated?
+        ::File.exist?('/Library/Application Support/Apple/Remote Desktop/RemoteManagement.launchd')
+      end
+
+      def configured?
+        RemoteManagement.plist.include?('ARD_AllLocalUsers = true') &&
+          RemoteManagement.plist.include?('ARD_AllLocalUsersPrivs = 1073742079') &&
+          RemoteManagement.plist.include?(full_privileges)
+      end
+
+      def plist
```

Comment:
Could we name this method `plist_contents`? Current name sounds like it's returning the location of the plist file itself, when in reality, it's returning the plist contents. It would make the implementation of the `configured?` a bit clearer, also.